libnopegl/src/backends/gl/rendertarget_gl.c

```diff
@@ -94,15 +94,15 @@ static int create_fbo(struct rendertarget *s, int resolve, GLuint *idp)
     const struct rendertarget_params *params = &s->params;
 
     GLuint id = 0;
-    int nb_color_attachments = 0;
+    uint32_t nb_color_attachments = 0;
 
     ngli_glGenFramebuffers(gl, 1, &id);
     ngli_glBindFramebuffer(gl, GL_FRAMEBUFFER, id);
 
     for (size_t i = 0; i < params->nb_colors; i++) {
         const struct attachment *attachment = &params->colors[i];
         const struct texture *texture = resolve ? attachment->resolve_target : attachment->attachment;
-        const int layer = resolve ? attachment->resolve_target_layer : attachment->attachment_layer;
+        const uint32_t layer = resolve ? attachment->resolve_target_layer : attachment->attachment_layer;
```

Comment:
the gl functions takes an `int` for the layer

Reply:
Commit dropped.